Sulakore/Habbo/Packages/HUserObject.cs

```diff
@@ -43,7 +43,9 @@ public HUserObject(HPacket packet)
             ScratchesLeft = packet.ReadInt32();
 
             StreamPublishingAllowed = packet.ReadBoolean();
-            LastAccess = DateTime.Parse(packet.ReadUTF8());
+
+            if (DateTime.TryParse(packet.ReadUTF8(), out DateTime _lastAccess)) //only save if parsing was successful to not save a default DateTime value
```

Comment:
Nit: Only private fields are prefixed with underscore.  Change `_lastAccess` to `lastAccess`.